### PR TITLE
added elasticsearch query util

### PR DESCRIPTION
**Pull request by haraprasadj**

### CODE REVIEW

File: suites/elasticTest.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const { expect } = require('chai');
+const { search } = require('../utils/elasticsearch.js');
+
+const index = 'etl_mickey_1';
+const query = '{ "query": { "nested": { "path": "BCMAMedicationLog", "query": { "range": { "BCMAMedicationLog.DosesOrdered": { "lt": 0.0029 } } } } } }';
+
+// Sample scenarios to query elasticsearch and validate results
+// scenario marked as manual to prevent it from running in CI pipeline
+Scenario('check length', async () => {
```
Comment (themarcelor) [inline suites/elasticTest.js]: Should we favour this new `.tag` chained function over the old way of tagging?

File: utils/elasticsearch.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const { Client } = require('@elastic/elasticsearch');
+const { Bash } = require('./bash');
+
+const service = 'aws-es-proxy-deployment';
+const bash = new Bash();
+const ES_URL = 'http://localhost:9200';
+
+module.exports = {
+  search: async (index, query) => {
+    const res = bash.runCommand(`g3kubectl port-forward deployment/${service} 9200:9200 &`);
```
Comment (themarcelor) [inline utils/elasticsearch.js]: Maybe get a +1 from DevOps to gather caveats on potential accumulation of kubectl port-forward processes in the background? 🤷‍♂

File: suites/elasticTest.js (hunk 1 above)
Comment (haraprasadj) [reply suites/elasticTest.js]: I thought it looked nice :)

File: suites/elasticTest.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Feature('es');
+
+const { expect } = require('chai');
+const { search } = require('../utils/elasticsearch.js');
+
+const index = 'etl_mickey_1';
```
Comment (themarcelor) [inline suites/elasticTest.js]: Was that supposed to be hardcoded?

File: utils/elasticsearch.js (hunk 2 above)
Comment (haraprasadj) [reply utils/elasticsearch.js]: it should only run once - you should get an error if you retry - Error listen tcp6 [::1]:9200: bind: address already in use]

File: suites/elasticTest.js (hunk 1 above)
Comment (haraprasadj) [reply suites/elasticTest.js]: this sample test is temporary anyway

File: utils/elasticsearch.js (hunk 2 above)
Comment (themarcelor) [reply utils/elasticsearch.js]: Concurrent PR checks + potential executions from local terminals can create multiple instances of the same command, right?

File: suites/elasticTest.js (hunk 3 above)
Comment (haraprasadj) [reply suites/elasticTest.js]: Again, sample test ... once we start using it, we can remove this

File: utils/elasticsearch.js (hunk 2 above)
Comment (haraprasadj) [reply utils/elasticsearch.js]: The command will fail to run a second time.

**Pull request merged**